training/prep_test_env.py

```diff
@@ -0,0 +1,61 @@
+#
```

Comment:
I mean this file can be deleted - as you already have `training/utils/prepare_test_environment.py` added in this PR